edgexfoundry/app-rfid-llrp-inventory
# refactor: Consume MakeItRun rename to Run

**If your build fails** due to your commit message not passing the build checks, please review the guidelines here: https://github.com/edgexfoundry/app-rfid-llrp-inventory/blob/main/.github/Contributing.md

## PR Checklist
Please check if your PR fulfills the following requirements:

https://github.com/edgexfoundry/edgex-docs/pull/1060

## Testing Instructions
Run non-secure EdgeX
Build App LLRP
Run App LLRP with `-cp -d`
Verify service starts up and is waiting for messages

## New Dependency Instructions (If applicable)
